ca/certificate-authority.go

```diff
@@ -156,7 +120,11 @@ func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config
 		return nil, err
 	}
 
-	pa := policy.NewPolicyAuthorityImpl()
+	pa, err := policy.NewPolicyAuthorityImpl(commonConfig)
```

Comment:
Let's pass the PolicyAuthority into this function instead

Then we don't have to pass around that opaque config object and we'll know exactly what this function requires